tests/html5skin-test.js

```diff
@@ -352,6 +352,8 @@ describe('Controller', function() {
       controller.onSetEmbedCode('newEmbedCode');
       controller.onEmbedCodeChanged('newEmbedCode');
       expect(controller.skin.state.currentPlayhead).toBe(0);
+      controller.toggleMultiAudioScreen();
```

Comment:
This test checks whether the playhead is reset on embed code change, so I don't  think this is the right place for these checks. Can we create a separate test in the multi audio section and check that toggling the screen on and off works correctly?

Reply:
\+